CASSANDRA-17264 - Content Update, Link fixes

Paul-TT opened the pull request:
Fixing various link errors that were flagged in site wide link checker. https://lists.apache.org/thread/3qtxndr4bxw5xo9gshrl4t2vyk4ym8o1

Patch by Paul: For CASSANDRA-17264

Paul-TT committed:
CASSANDRA-17264 - Content Update, Link fixes

ErickRamirezAU commented on site-content/source/modules/ROOT/pages/blog/Apache-Cassandra-4.0-Overview.adoc:
```diff
@@ -21,7 +21,7 @@ Testing included purpose-built new tools to cover every requirement:
 * Unit / dtest coverage expansion
 
-Those {site-url}blog/Testing-Apache-Cassandra-4.html[tools] were perfected and deployed to help meet quality goals and set a baseline for any future version of Cassandra. They provide needed infrastructure to ensure future releases retain the highest levels of quality and correctness. 
+Those {site-url}_/blog/Testing-Apache-Cassandra-4.html[tools] were perfected and deployed to help meet quality goals and set a baseline for any future version of Cassandra. They provide needed infrastructure to ensure future releases retain the highest levels of quality and correctness. 
```
```suggestion
Those xref:blog/Testing-Apache-Cassandra-4.html[tools] were perfected and deployed to help meet quality goals and set a baseline for any future version of Cassandra. They provide needed infrastructure to ensure future releases retain the highest levels of quality and correctness. 

```

ErickRamirezAU commented on site-content/source/modules/ROOT/pages/blog/Apache-Cassandra-Changelog-1-October-2020.adoc:
```diff
@@ -12,15 +12,15 @@ Our monthly roundup of key activities and knowledge to keep the community inform
 === Added
 
-With CASSANDRA-15013, the community improved Cassandra's ability to xref:blog/improving-resiliency.adoc[handle high throughput workloads], while having enough safeguards in place to protect itself from potentially going out of memory.
+With CASSANDRA-15013, the community improved Cassandra's ability to xref:blog/Apache-Cassandra-Changelog-1-October-2020.adoc[handle high throughput workloads], while having enough safeguards in place to protect itself from potentially going out of memory.
```
Isn't this supposed to link to the "front door & backpressure" blog post?

```suggestion
With CASSANDRA-15013, the community improved Cassandra's ability to xref:blog/Improving-Apache-Cassandras-Front-Door-and-Backpressure.adoc[handle high throughput workloads], while having enough safeguards in place to protect itself from potentially going out of memory.

```

ErickRamirezAU commented on site-content/source/modules/ROOT/pages/blog/Apache-Cassandra-Changelog-2-December-2020.adoc:
```diff
@@ -14,10 +14,11 @@ Our monthly roundup of key activities and knowledge to keep the community inform
 
 === Changed
 
+https://lists.apache.org/list.html?\dev@cassandra.apache.org:2021-08[JKD 17, window=_blank]
```
I'm not sure what this line is for:
- the link doesn't exist
- I don't know what "JKD 17" is. Is that meant to be `JDK 17`?
In any case, I don't think this belongs here so should be removed.

```suggestion
```

Paul-TT replied:
Hey Erik,  that resolves as https://lists.apache.org/list.html?dev@cassandra.apache.org:2021-08 after processing.  the "\" after the "?" is necessary to render the link properly.  I confirmed that the link resolves when you do a build.

ErickRamirezAU replied:
But what is it a link to? I mean, what information is being provided for this line?

The link goes to the Dev mailing list archive for August last year but the purpose for linking to the mailing list isn't clear to me. 🙂

michaelsembwever replied:
Agree, what is the purpose or value of linking to this page…?
<img width="1378" alt="Screenshot 2022-01-29 at 13 15 20" src="https://user-images.githubusercontent.com/559444/151660634-0b1e6a69-a7ff-40af-bea9-d22703fb86a9.png">

Paul-TT replied:
I see. Yeah, I think that can just be removed.

ErickRamirezAU commented on site-content/source/modules/ROOT/pages/blog/Apache-Cassandra-Changelog-6-April-2021.adoc:
```diff
@@ -33,11 +33,11 @@ The https://lists.apache.org/thread.html/r1e545f41faf6c4ac0a4c196f9ae85a142abd89
 === Added
 
-As the community gets closer to the launch of 4.0, we are organizing a celebration with the help of ASF – xref:blog/world_party.adoc[Cassandra World Party 4.0,window=_blank] will be a one-day, no-cost virtual event on Wednesday, April 28 to bring the global community together in celebration of the upcoming release milestone. The https://sessionize.com/cassandra[CFP for 5-minute lightning talks,wi ...
+As the community gets closer to the launch of 4.0, we are organizing a celebration with the help of ASF – xref:blog/World_Party.adoc[Cassandra World Party 4.0,window=_blank] will be a one-day, no-cost virtual event on Wednesday, April 28 to bring the global community together in celebration of the upcoming release milestone. The https://sessionize.com/cassandra[CFP for 5-minute lightning talks,wi ...
```
This link doesn't render correctly. I'm new to Antora so I'm asking around to figure out the correct code for this. 🙂

ErickRamirezAU replied:
It's a typo in the filename -- needs a dash (`-`) instead of an underscore (`_`).

```suggestion
As the community gets closer to the launch of 4.0, we are organizing a celebration with the help of ASF – xref:blog/World-Party.adoc[Cassandra World Party 4.0,window=_blank] will be a one-day, no-cost virtual event on Wednesday, April 28 to bring the global community together in celebration of the upcoming release milestone. The https://sessionize.com/cassandra[CFP for 5-minute lightning talks,window=_blank] is open now until April 9 – newcomers welcome! https://hopin.com/events/apache-cassandra-4-0-world-party[Register here,window=_blank].
```

ErickRamirezAU commented on site-content/source/modules/ROOT/pages/blog/Cassandra-and-Kubernetes-SIG-Update-and-Survey.adoc:
```diff
@@ -60,4 +60,4 @@ image::blog/single-region-multi-datacenter.png[Multiple Workloads / Datacenters
 
-Join the https://cwiki.apache.org/confluence/display/CASSANDRA/Cassandra+Kubernetes+Operator+SIG[biweekly meetings,window=_blank] to stay informed.
\ No newline at end of file
+Join the https://cwiki.apache.org/confluence/display/CASSANDRA/Cassandra+Kubernetes+SIG[biweekly meetings,window=_blank] to stay informed.
```
Use the permanent link so even when the page title changes, the link will continue to work:

```suggestion
Join the https://cwiki.apache.org/confluence/x/GCzcC[biweekly meetings,window=_blank] to stay informed.
```

ErickRamirezAU commented on site-content/source/modules/ROOT/pages/blog/Finding-Bugs-in-Cassandra's-Internals-with-Property-based-Testing.adoc:
```diff
@@ -1,12 +1,12 @@
-= Finding Bugs in Cassandra's Internals with Property-based Testing
+= Finding Bugs in Cassandra\s Internals with Property-based Testing
```
Is this supposed to be an "escaped apostrophe"?

```suggestion
= Finding Bugs in Cassandra\'s Internals with Property-based Testing
```

ErickRamirezAU commented on site-content/source/modules/ROOT/pages/blog/Finding-Bugs-in-Cassandra's-Internals-with-Property-based-Testing.adoc:
```diff
@@ -1,12 +1,12 @@
 :keywords: 
 
-As of September 1st, the Apache Cassandra community has shifted the focus of Cassandra 4.0 development from new feature work to testing, validation, and hardening, with the goal of releasing a stable 4.0 that every Cassandra user, from small deployments to large corporations, can deploy with confidence. There are several projects and methodologies that the community is undertaking to this end. On ...
+As of September 1st, the Apache Cassandra community has shifted the focus of Cassandra 4.0 development from new feature work to testing, validation, and hardening, with the goal of releasing a stable 4.0 that every Cassandra user, from small deployments to large corporations, can deploy with confidence. There are several projects and methodologies that the community is undertaking to this end. On ...
```
The link wasn't coded correctly:

```suggestion
As of September 1st, the Apache Cassandra community has shifted the focus of Cassandra 4.0 development from new feature work to testing, validation, and hardening, with the goal of releasing a stable 4.0 that every Cassandra user, from small deployments to large corporations, can deploy with confidence. There are several projects and methodologies that the community is undertaking to this end. One of these is the adoption of property-based testing, which was xref:blog/Testing-Apache-Cassandra-4.adoc[previously introduced here,window=_blank]. This post will take a look at a specific use of this approach and how it found a bug in a new feature meant to ensure data integrity between the client and Cassandra.
```

ErickRamirezAU commented on site-content/source/modules/ROOT/pages/blog/Introducing-Apache-Cassandra-4-Beta-Battle-Tested-From-Day-One.adoc:
```diff
@@ -8,7 +8,7 @@
 This is the most *stable* Apache Cassandra in history; you should start using Apache Cassandra 4.0 Beta today in your test and QA environments, head to the xref:download.adoc[downloads site] to get your hands on it. The Cassandra community is on a mission to deliver a 4.0 GA release that is ready to be deployed to production. You can guarantee this holds true by running your application workloads ...
 
-With over 1,000 bug fixes, improvements and new features and the project’s wholehearted xref:blog/testing_apache_cassandra.adoc[focus on quality] with xref:blog/finding_bugs_with_property_based_testing.adoc[replay, fuzz, property-based, fault-injection], and performance tests on clusters as large as 1,000 nodes and with hundreds of real world use cases and schemas tested, Cassandra 4.0 redefines  ...
+With over 1,000 bug fixes, improvements and new features and the project’s wholehearted xref:blog/Testing-Apache-Cassandra-4.adoc[focus on quality] with xref:blog/Finding-Bugs-in-Cassandra's-Internals-with-Property-based-Testing.adoc[replay, fuzz, property-based, fault-injection], and performance tests on clusters as large as 1,000 nodes and with hundreds of real world use cases and schemas teste ...
```
Need to escape the apostrophe in the filename:

```suggestion
With over 1,000 bug fixes, improvements and new features and the project’s wholehearted xref:blog/Testing-Apache-Cassandra-4.adoc[focus on quality] with xref:blog/Finding-Bugs-in-Cassandra\'s-Internals-with-Property-based-Testing.adoc[replay, fuzz, property-based, fault-injection], and performance tests on clusters as large as 1,000 nodes and with hundreds of real world use cases and schemas tested, Cassandra 4.0 redefines what users should expect from any open or closed source database. With software, hardware, and QA testing donations from the likes of Instaclustr, iland, Amazon, and Datastax, this release has seen an unprecedented cross-industry collaboration towards releasing a battle-tested database with enterprise security features and an understanding of what it takes to deliver scale in the cloud.
```

ErickRamirezAU commented on site-content/source/modules/ROOT/pages/blog/Join-Cassandra-GSoC-2021.adoc:
```diff
@@ -17,7 +17,7 @@ The Apache Software Foundation (ASF) has been a https://community.apache.org/gso
 
 ==== Getting Started
-The best way to get started if you’re new to Apache Cassandra is to get acquainted by reading the {site-url}/doc/latest/[documentation,window=_blank] and setting up a local {site-url}/doc/latest/development/ide.html[development environment,window=_blank]. Play around with a locally running instance via {site-url}/doc/latest/tools/cqlsh.html?highlight=cqlsh[cqlsh,window=_blank] and {site-url}/doc/ ...
+The best way to get started if you’re new to Apache Cassandra is to get acquainted by reading the {site-url}doc/latest/[documentation,window=_blank] and setting up a local {site-url}doc/latest/development/ide.html[development environment,window=_blank]. Play around with a locally running instance via {site-url}doc/latest/tools/cqlsh.html?highlight=cqlsh[cqlsh,window=_blank] and {site-url}doc/late ...
```
The links with relative paths aren't working so I've had to hardcode one of them and update the others.

**EDIT:** Hardcoding not required, works with `link:` macro.

```suggestion
The best way to get started if you’re new to Apache Cassandra is to get acquainted by reading the link:/doc/latest/[documentation,window=_blank] and setting up a local xref:development/ide.adoc[development environment,window=_blank]. Play around with xref:quickstart.adoc[a locally running instance using Docker,window=_blank] to get a feel for how to use the database. If you run into problems or roadblocks during this exercise, don’t be shy to ask questions via the xref:community.adoc[community channels] like the developers mailto:dev-subscribe@cassandra.apache.org[mailing list] or the #cassandra-dev channel on the https://s.apache.org/slack-invite[ASF Slack,window=_blank].
```

ErickRamirezAU commented on site-content/source/modules/ROOT/pages/development/documentation.adoc:
```diff
@@ -8,7 +8,7 @@
 repository.
 We use a static site generator, http://www.antora.org/[Antora], to create pages hosted at
-{site-url}/doc/latest/[cassandra.apache.org].
+{site-url}doc/latest/[cassandra.apache.org].
```
```suggestion
link:/doc/latest/[cassandra.apache.org].
```

Paul-TT replied:
Mick had requested the removal of all hard coded absolute paths of 'https://cassandra.apache.org'

{site-url} will be processed as https://cassandra.apache.org/'

I can confirm locally that it points to the right place.

michaelsembwever replied:
that's correct – we don't want to hardcode ` https://cassandra.apache.org/` anywhere. (one example: it needs to work the same on  https://cassandra.staged.apache.org/ )

ErickRamirezAU replied:
Nope, definitely doesn't work. 🙂

`{site-url}` doesn't get rendered at all. This is what it looks like in both [production](https://cassandra.apache.org/_/development/documentation.html) and [staging](https://cassandra.staged.apache.org/_/development/documentation.html):

```
The official Cassandra documentation lives in the project’s git repository. \
  We use a static site generator, Antora, to create pages hosted at //doc/latest/[cassandra.apache.org].
```

I've spent 2 hours last week just on this single issue and couldn't find any other way of making it work. I finally spoke to the Antora experts and they confirmed that since the Doc pages are not asciidoc, Antora doesn't know how to render `{site-url}` and so the link to the docs have to be hardcoded.

Feel free to have a look at http://18.192.24.126:9000/_/development/documentation.html where I have your PR built/staged. Cheers! 🍻

michaelsembwever replied:
> {site-url} doesn't get rendered at all. This is what it looks like in both production and staging:

We are still manually mangling the pages before they hit staging and prod.
See https://github.com/apache/cassandra-builds/compare/trunk...thelastpickle:mck/16765#diff-1f760d89dfb81ea05a11d862007a4e4e586b38aac7ae87f62c5b2e86571809c0R1288-R1290

So `//doc/latest/` will get rewritten to `/doc/latest/`

But there's a separate problem here… it doesn't appear as a link…
Should it be like…?
```
We use a static site generator, http://www.antora.org/[Antora], to create pages hosted at
link:doc/latest/[cassandra.apache.org].
```

ErickRamirezAU replied:
I've updated the revision above to:

```
link:/doc/latest/[cassandra.apache.org].
```

ErickRamirezAU replied:
BTW the link on line 22 will also need to be fixed:

```
Feel free to take a look at http://cassandra.apache.org/docs[existing documents] to get a better idea how we structure our documents.
```

ErickRamirezAU commented on site-content/source/modules/ROOT/pages/development/documentation.adoc:
```diff
@@ -86,7 +86,7 @@ _Recommended for major changes_
 Significant changes to the documentation are best managed through our
 Jira issue tracker. Please follow the same
-{site-url}/doc/latest/development/patches.html[contribution
+{site-url}doc/latest/development/patches.html[contribution
```
```suggestion
xref:development/patches.adoc[contribution
```

ErickRamirezAU commented on site-content/source/modules/ROOT/pages/development/gettingstarted.adoc:
```diff
@@ -24,7 +24,7 @@ Here are some suggestions for ways to contribute:
 repository along with the Cassandra code base.
 To submit changes to the documentation, follow the standard process for
-submitting a xref::patches.adoc[patch].
+submitting a xref::development/patches.adoc[patch].
```
```suggestion
submitting a xref:development/patches.adoc[patch].
```

ErickRamirezAU commented on site-content/source/modules/ROOT/pages/development/gettingstarted.adoc:
```diff
@@ -65,4 +65,4 @@ coverage, as with most code bases.
 enhancing the existing test coverage.
 You'll both improve your knowledge of the code before you start on an
-enhancement, and reduce the chance introducing issues with your change. See xref::testing.adoc[testing] and xref::patches.adoc[patches] for more detail.
+enhancement, and reduce the chance introducing issues with your change. See xref::development/testing.adoc[testing] and xref::development/patches.adoc[patches] for more detail.
```
```suggestion
enhancement, and reduce the chance introducing issues with your change. See xref:development/testing.adoc[testing] and xref:development/patches.adoc[patches] for more detail.
```

ErickRamirezAU commented on site-content/source/modules/ROOT/pages/quickstart.adoc:
```diff
@@ -165,7 +165,7 @@ Hey, that wasn’t so hard, was it?
 
 * Read through the xref:cassandra-basics.adoc[Cassandra Basics] to learn main concepts and how Cassandra works at a high level.
-* To understand Cassandra in more detail, head over to the xref:docs[Docs].
+* To understand Cassandra in more detail, head over to the xref:Cassandra::index.adoc[Docs].
```
```suggestion
* To understand Cassandra in more detail, head over to the https://cassandra.apache.org/doc/latest/[Docs].
```

michaelsembwever replied:
no self domain name please.

ErickRamirezAU reviewed (changes requested):
@Paul-TT please review the changes I've suggested. Cheers! 🍻

Paul-TT committed:
Update site-content/source/modules/ROOT/pages/blog/Apache-Cassandra-4.0-Overview.adoc

Co-authored-by: Erick Ramirez <erickramirezau@gmail.com>

Paul-TT committed:
Update site-content/source/modules/ROOT/pages/blog/Apache-Cassandra-Changelog-1-October-2020.adoc

I had gotten some guidance on where to point certain links but you know the content better than i do.  So I'll stick to your decision

Co-authored-by: Erick Ramirez <erickramirezau@gmail.com>

Paul-TT committed:
Update site-content/source/modules/ROOT/pages/blog/Apache-Cassandra-Changelog-6-April-2021.adoc

Co-authored-by: Erick Ramirez <erickramirezau@gmail.com>

Paul-TT committed:
Update site-content/source/modules/ROOT/pages/blog/Cassandra-and-Kubernetes-SIG-Update-and-Survey.adoc

Co-authored-by: Erick Ramirez <erickramirezau@gmail.com>

Paul-TT committed:
Update site-content/source/modules/ROOT/pages/blog/Apache-Cassandra-Changelog-2-December-2020.adoc

accept change

Co-authored-by: Erick Ramirez <erickramirezau@gmail.com>

Paul-TT committed:
Update site-content/source/modules/ROOT/pages/development/gettingstarted.adoc

Co-authored-by: Erick Ramirez <erickramirezau@gmail.com>

Paul-TT committed:
Update site-content/source/modules/ROOT/pages/development/gettingstarted.adoc

Co-authored-by: Erick Ramirez <erickramirezau@gmail.com>

Paul-TT committed:
Update site-content/source/modules/ROOT/pages/quickstart.adoc

Co-authored-by: Erick Ramirez <erickramirezau@gmail.com>

Paul-TT committed:
Update site-content/source/modules/ROOT/pages/development/documentation.adoc

Co-authored-by: Erick Ramirez <erickramirezau@gmail.com>

Paul-TT committed:
Update site-content/source/modules/ROOT/pages/development/documentation.adoc

Co-authored-by: Erick Ramirez <erickramirezau@gmail.com>

Paul-TT committed:
Update site-content/source/modules/ROOT/pages/blog/Finding-Bugs-in-Cassandra's-Internals-with-Property-based-Testing.adoc

Co-authored-by: Erick Ramirez <erickramirezau@gmail.com>

Paul-TT committed:
Update site-content/source/modules/ROOT/pages/blog/Finding-Bugs-in-Cassandra's-Internals-with-Property-based-Testing.adoc

Co-authored-by: Erick Ramirez <erickramirezau@gmail.com>

Paul-TT committed:
Update site-content/source/modules/ROOT/pages/blog/Introducing-Apache-Cassandra-4-Beta-Battle-Tested-From-Day-One.adoc

Co-authored-by: Erick Ramirez <erickramirezau@gmail.com>

Paul-TT committed:
Update site-content/source/modules/ROOT/pages/blog/Join-Cassandra-GSoC-2021.adoc

Co-authored-by: Erick Ramirez <erickramirezau@gmail.com>

ErickRamirezAU commented on site-content/source/modules/ROOT/pages/blog/Join-Cassandra-GSoC-2021.adoc:
```diff
@@ -17,7 +17,7 @@ The Apache Software Foundation (ASF) has been a https://community.apache.org/gso
 
 ==== Getting Started
-The best way to get started if you’re new to Apache Cassandra is to get acquainted by reading the {site-url}/doc/latest/[documentation,window=_blank] and setting up a local {site-url}/doc/latest/development/ide.html[development environment,window=_blank]. Play around with a locally running instance via {site-url}/doc/latest/tools/cqlsh.html?highlight=cqlsh[cqlsh,window=_blank] and {site-url}/doc/ ...
+The best way to get started if you’re new to Apache Cassandra is to get acquainted by reading the https://cassandra.apache.org/doc/latest/[documentation,window=_blank] and setting up a local xref:development/ide.adoc[development environment,window=_blank]. Play around with xref:quickstart.adoc[a locally running instance using Docker,window=_blank] to get a feel for how to use the database. If you ...
```
I'm sorry I made a mistake in my previous review. The URL for `/doc/latest` doesn't need to be hardcoded, just needs the `link:` macro.

```suggestion
The best way to get started if you’re new to Apache Cassandra is to get acquainted by reading the link:/doc/latest/[documentation,window=_blank] and setting up a local xref:development/ide.adoc[development environment,window=_blank]. Play around with xref:quickstart.adoc[a locally running instance using Docker,window=_blank] to get a feel for how to use the database. If you run into problems or roadblocks during this exercise, don’t be shy to ask questions via the xref:community.adoc[community channels] like the developers mailto:dev-subscribe@cassandra.apache.org[mailing list] or the #cassandra-dev channel on the https://s.apache.org/slack-invite[ASF Slack,window=_blank].
```

ErickRamirezAU commented on site-content/source/modules/ROOT/pages/community.adoc:
```diff
@@ -331,7 +331,7 @@ https://www.apache.org/theapacheway/index.html[The Apache Way,window=blank]
 [openblock, float75 full-800]
 ----------
-Contributors are individuals who contribute patches—source code, documentation, help on mailing lists, website—to Apache projects. While contributors do not have a specific governance role, they are crucial to the project’s success. Read the docs to learn how to {site-url}doc/latest/development/index.html[contribute to Cassandra,window=blank], and review our https://cwiki.apache.org/confluence/di ...
+Contributors are individuals who contribute patches—source code, documentation, help on mailing lists, website—to Apache projects. While contributors do not have a specific governance role, they are crucial to the project’s success. Read the docs to learn how to {site-url}doc/latest/development/index.html[contribute to Cassandra,window=blank], and review our https://cwiki.apache.org/confluence/di ...
```
There’s a merge conflict with this page which needs to be resolved:

```
<<<<<<< trunk
Contributors are individuals who contribute patches—source code, documentation, help on mailing lists, website—to Apache projects. While contributors do not have a specific governance role, they are crucial to the project’s success. Read the docs to learn how to {site-url}doc/latest/development/index.html[contribute to Cassandra,window=blank], and review our https://cwiki.apache.org/confluence/display/CASSANDRA/Cassandra+Project+Governance[governance,window=blank] page to understand how we vote on code contributions.
=======
Contributors are individuals who contribute patches—source code, documentation, help on mailing lists, website—to Apache projects. While contributors do not have a specific governance role, they are crucial to the project’s success. Read the docs to learn how to link:{site-url}_/development/index.html[contribute to Cassandra,window=blank], and review our https://cwiki.apache.org/confluence/display/CASSANDRA/Apache+Cassandra+Project+Governance[governance,window=blank] page to understand how we vote on code contributions.
>>>>>>> trunk
```

It should be this:

```suggestion
Contributors are individuals who contribute patches—source code, documentation, help on mailing lists, website—to Apache projects. While contributors do not have a specific governance role, they are crucial to the project’s success. Read the docs to learn how to xref:/development/index.adoc[contribute to Cassandra,window=blank], and review our https://cwiki.apache.org/confluence/display/CASSANDRA/Cassandra+Project+Governance[governance,window=blank] page to understand how we vote on code contributions.
```

ErickRamirezAU reviewed (changes requested):
I’m sorry I made a mistake in one of my suggestions. I’ve corrected it now and needs to get committed.

There’s also a merge conflict with the page `community.adoc` which needs to be resolved. Cheers! 🍻 

/cc @dtopdontstop

ErickRamirezAU committed:
Update site-content/source/modules/ROOT/pages/blog/Join-Cassandra-GSoC-2021.adoc

ErickRamirezAU committed:
Update site-content/source/modules/ROOT/pages/community.adoc

Paul-TT committed:
Merge branch 'trunk' into trunk

Paul-TT commented:
Hi Erik.  I think I'm done.  Apologies in advance if I'm missing something.  Let me know if there are more steps I need to take to close this out.

michaelsembwever commented:
Please don't `Merge branch 'trunk' into trunk`. Development (feature) branches are best kept up to date by rebasing them, not by merging new trunk changes in. (Merging should only be done in one-direction, and that's from development/feature branches into release/trunk branches.

michaelsembwever commented:
Merged manually with https://github.com/apache/cassandra-website/commit/2c535052aa6ccba4af32ba72dbc986cae2938a02
